jdaviz/core/template_mixin.py

```diff
@@ -767,7 +766,7 @@ def not_from_plugin(data):
             return data.meta.get('Plugin', None) is None
 
         def not_from_plugin_model_fitting(data):
-            return data.meta.get('Plugin', None) != 'model-fitting'
+            return data.meta.get('Plugin', None) != 'ModelFitting'
```

Comment:
switched from manual meta-data tags for the plugins to using the class name, so that we can use these programmatically within filters as well.